### ISSUE TITLE
Spawn point

### ISSUE THREAD

**Issue by Tilepaper** [toolbox4minecraft/amidst#307]
Sometimes, especially on random seeds, Amidst will put the spawn point at the origin (0,0), even if this point is in the sea or somewhere unreasonable.

**Comment by stefandollase** [comment]
Indeed, we currently only use a very simple heuristic to determine the location of the world spawn point, see [here](https://github.com/toolbox4minecraft/amidst/blob/master/src/main/java/amidst/mojangapi/world/oracle/HeuristicWorldSpawnOracle.java).

I marked this issue as a feature request to improve this heuristic. The Minecraft Wiki may contain information that allows us to calculate the world spawn more precisely.
